pkg/telemetry/telemetry.go

```diff
@@ -65,7 +58,9 @@ func NewTelemetryEngine(conn pgxconn.PgxConn, uuid [16]byte) (Telemetry, error)
 func (t *engine) writeMetadata() error {
 	promscale, err := promscaleMetadata()
 	if err != nil {
-		return fmt.Errorf("promscale metadata: %w", err)
+		// Err due to Uname failed to fetch. Send the metadata that's left without Uname
+		// and simply debug log with rate-limit.
```

Comment:
Comment is too specific and refers to underlying implementation. Since the implementation changes over time, it's likely this comment will be wrong in the future. Probably best to remove it since it doesn't add much here.